.gitignore

```diff
@@ -5,4 +5,4 @@ __pycache__/
 *.egg-info
 /build
 /dist
-/flake8_executable/_version.py
```

Comment:
Maybe it's easier to do this way: Let's take out one commit per time and make it into a PR since it might not be easy to stack PRs in this case.

Let's start from an easy one. What's the reason to delete this line?

Reply:
Now I see this change wasn't necessary, in the PR I've now used [setuptools_scm](https://github.com/pypa/setuptools_scm)'s pyproject.toml config